DependencyInjection/GesdinetJWTRefreshTokenExtension.php

```diff
@@ -51,7 +52,8 @@ public function load(array $configs, ContainerBuilder $container): void
         $refreshTokenClass = RefreshTokenEntity::class;
         $objectManager = 'doctrine.orm.entity_manager';
 
-        if (!class_exists(DoctrineOrmMappingsPass::class) || 'mongodb' === strtolower($config['manager_type'])) {
+        // Change the refresh token and object manager to the MongoDB ODM if the configuration explicitly sets it or if the ORM is not installed and the MongoDB ODM is
+        if ('mongodb' === strtolower($config['manager_type']) || (!class_exists(EntityManager::class) && class_exists(DocumentManager::class))) {
```

Comment:
Changing this to use the object managers instead of the compiler passes in the bundles allows to break the dev dependencies on the bundles (which, were only pulled in for these checks).